cloud-core/src/main/java/cloud/commandframework/execution/FilteringCommandSuggestionProcessor.java

```diff
@@ -46,12 +98,19 @@
         if (context.getInputQueue().isEmpty()) {
             input = "";
         } else {
-            input = context.getInputQueue().peek();
+            input = String.join(" ", context.getInputQueue());
         }
+        final @Nullable String inputLower = this.ignoreCase ? input.toLowerCase(Locale.ENGLISH) : null;
```

Comment:
I believe both here and a bit below, what you're looking for is `toLowerCase(Locale.ROOT)`, for language-insensitive to-lower-case